docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,114 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
 ============
 
 |===
+
+[float]
+[[audit-logging-settings]]
+===== Audit logging settings
+
+You can enable audit logging to support compliance, accountability and security. When enabled {kib} will capture:
+
+- Who performed an action
+- What action was performed
+- When it occurred
+
+For more details and a reference of audit events see <<xpack-security-audit-logging>>.
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.enabled`
+| Set to `true` to enable audit logging for security events. *Default:* `false`
+|===
+
+[float]
+[[audit-logging-settings-beta]]
+===== Audit logging settings (beta)
+
+beta[]
+
+To enable the new audit logger (beta) you need to specify where the new audit
```

Comment:
Could we shorten this? For example:

To enable the audit logger, specify where you want to write the audit events using `xpack.security.audit.appender`.

Reply:
Thanks for all your suggestions, looking much better now :)